spec.html

```diff
@@ -24706,27 +25391,36 @@ <h1>
       <h1>
         Runtime Semantics: ClassElementEvaluation (
           _object_: unknown,
-        ): either a normal completion containing either a ClassFieldDefinition Record, a ClassStaticBlockDefinition Record, a Private Name, or ~unused~, or an abrupt completion
+          _metadataList: a List of Metadata Records,
```

Comment:
```suggestion
          _metadataList_: a List of Metadata Records,
```
https://github.com/tc39/ecma262/pull/2417/files#r800188546